include/nuttx/timers/oneshot.h

```diff
@@ -283,7 +286,10 @@ int oneshot_start(FAR struct oneshot_lowerhalf_s *lower,
 {
   clock_t tick;
 
-  DEBUGASSERT(lower->ops->tick_start);
```

Comment:
need include errno.h

Reply:
errno.h already there for oneshot.h, the timer.h added a errno.h, let's see if CI will report some problem.